dll/win32/browseui/addressband.h

```diff
@@ -50,6 +50,8 @@ class CAddressBand :
 private:
     void FocusChange(BOOL bFocus);
     void CreateGoButton();
+    WCHAR GetFocusKey();
+    WCHAR m_chFocusKey;
```

Comment:
Please move the member up with the other ones.

Reply:
OK.